README.md

```diff
@@ -46,6 +47,81 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script does deploy a simple order market maker order and a relevant
+```
+// Get token id of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+//export the result
+export TOKEN_ID_DAI=[Result from last call]
+
+// Get token id of TrueUSD
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network rinkeby
+
+//export the result
+export TOKEN_ID_TUSD=[Result from last call]
+
+// Make sure there is enough funds in the exchange for the intended trades
```

Comment:
```suggestion
// Make sure there are enough funds in the exchange for the intended trades
```
Are we sure we don't want to make sure that "the users have deposited sufficient funds into the exchange"